src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,138 @@
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        printWithLine(List.of("Hello from\n" + logo));
+        printWithLine(List.of("Hello! I'm Duke", "What can I do for you?"));
+
+        String line;
+        Scanner in = new Scanner(System.in);
+        List<Task> items = new ArrayList<>();
+
+        do{
+            line = in.nextLine();
+            String keyword = line.split(" ")[0].toLowerCase();
+
+            try {
+                checkLineEmpty(line);
+                checkWord(line);
+                checkDescription(line);
+
+                if (line.equalsIgnoreCase("list")){
+                    printWithLine(List.of());
+                    List<String> messages = new ArrayList<>();
+                    System.out.println("   Here are the tasks in your list: ");
+                    for (int i = 0; i < items.size(); i++) {
+                        messages.add(i + 1 + "." + items.get(i));
+                    }
+                    printWithLine(messages);
+
+                } else if (line.split(" ")[0].equalsIgnoreCase("done")){
+                    printWithLine(List.of());
+                    Task markItem = items.get(Integer.parseInt(line.substring(5)) - 1);
+                    markItem.markAsDone();
+                    printWithLine(List.of("Nice! I've marked this task as done: ", " " + markItem));
+
+                } else if (line.split(" ")[0].equalsIgnoreCase("todo")){
+                    printWithLine(List.of());
+                    Task todoTask = new Todo(line.replace(line.split(" ")[0] + " ", ""));
+                    items.add(todoTask);
+                    printWithLine((List.of("Got it. I've added this task: ", todoTask.toString(), "Now you have " + items.size() + " task in the list. ")));
+
+                } else if (line.split(" ")[0].equalsIgnoreCase("deadline")){
+                    printWithLine(List.of());
+                    int position = line.indexOf("/");
+                    String time = line.split("/")[1].replace("by ", "");
+                    Task deadlineTask = new Deadline(line.substring(9, position - 1), time);
+                    items.add(deadlineTask);
+                    printWithLine((List.of("Got it. I've added this task: ", deadlineTask.toString(), "Now you have " + items.size() + " task in the list. ")));
+
+                } else if (line.split(" ")[0].equalsIgnoreCase("event")){
+                    printWithLine(List.of());
+                    int position = line.indexOf("/");
+                    String time = line.split("/")[1].replace("at ", "");
+                    Task eventTask = new Event(line.substring(6, position - 1), time);
+                    items.add(eventTask);
+                    printWithLine((List.of("Got it. I've added this task: ", eventTask.toString(), "Now you have " + items.size() + " task in the list. ")));
```

Comment:
Put inside a method to handle this else if block for readers to understand the flow clearly.

Reply:
Thanks for your advice, will update the code.